CHANGELOG.md

```diff
@@ -5,6 +5,15 @@ All notable changes to this project will be documented in this file.
 The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/)
 and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.html).
 
+## Unreleased
+
+### Changed
+
+- Service/network certificate validity period is no longer hardcoded and can instead be set by operators and renewed by members:
```

Comment:
I wonder if this is an opportunity to rename network certificate to service certificate? Or at least choose one and stick to it and it seems that we have referred to each one interchangeably.

Reply:
Done. This certificate should always be called service certificate from now on (rendering this PR rather large).